src/cli/env.rs

```diff
@@ -218,3 +218,9 @@ impl std::ops::Deref for BoolFlag {
         &self.0
     }
 }
+
+static IS_CI: std::sync::OnceLock<bool> = std::sync::OnceLock::new();
```

Comment:
We have `Env` in `env.rs` where you can add this. We shouldn't add random env vars around.